NEWS.md

```diff
@@ -590,6 +590,8 @@ Library improvements
   * `IOBuffer` can take the `sizehint` keyword argument to suggest a capacity of
     the buffer ([#25944]).
 
+  * Switched the order of return values in function `readdlm` when receiving `header=true` as a parameter, now the function returns a tuple with `(header, data)`. ([#25328])
```

Comment:
Please wrap lines at 92 characters. Also, this should be under "Breaking changes". Finally, I think we generally say "argument" rather than "parameter" (the latter being used for type parameters).

Reply:
My bad, I'll adjust accordingly.